Shared/Components/Cycle/CycleView.swift

```diff
@@ -42,3 +41,24 @@ struct CircleView: View {
         return date == nil ? "?" : formatter.string(from: date)
     }
 }
+
+//Liquid()
+//    .frame(width: 260, height: 260)
+//    .foregroundColor(colorSet.color1)
+//    .opacity(0.3)
+//
+//Liquid()
+//    .frame(width: 240, height: 240)
+//    .foregroundColor(colorSet.color2)
+//    .opacity(0.6)
```

Comment:
Oii pode deletar os códigos comentados se não for mais ser usado, se precisar dps bota 👍

Reply:
tá de boas.. vai ficar na minha branch se precisar.. então blz